### ISSUE TITLE
Supporting RTI data parsing

### ISSUE THREAD

**Issue by mbalabash** [aslushnikov/tracium#13]
Hello, according to [this message](https://github.com/GoogleChrome/puppeteer/pull/4591#issuecomment-502340394). Puppeteer supports RTI data in trace file with `--enable-thread-instruction-count` flag.

It will be nice if `Tracium` can parse this data if available.
I ready to make implementation, but I faced a [problem](https://github.com/puppeteer/puppeteer/issues/5223).

Could you provide some thoughts about this issue?

**Comment by mbalabash** [comment]
Guys (@aslushnikov @paulirish), did you try to get RTI data with docker before? 

I don't have `tidelta` in trace file (tried on node:10-slim and node:12-slim docker images).

**Comment by plesiecki** [comment]
Hello.
According to this https://calendar.perfplanet.com/2019/javascript-component-level-cpu-costs/ yet another flag is neeed, `--no-sandbox`. Can't confirm if it work or not.

**Comment by mbalabash** [comment]
@plesiecki Yes, this flag was in example too. I use it, but don't have `ticount` or `tidelta` in trace file (with chrome 77 and 78).

**Comment by erykpiast** [comment]
@stoyan could you provide an example, how to use this flag in the latest Chrome?

**Comment by mbalabash** [comment]
@erykpiast Thank your for attention to this problem.

I made some research about this issue. You can see [it here](https://github.com/puppeteer/puppeteer/issues/5223).

**Comment by erykpiast** [comment]
That's great, thank you! I didn't see that issue.

**Comment by mbalabash** [comment]
If you are using Linux distro [check this out](https://github.com/puppeteer/puppeteer/issues/5223#issuecomment-576448290) to help me with testing this issue.

**Comment by erykpiast** [comment]
I'm on MacOS unfortunately :(
